index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
```

Comment:
I'm considering auto generating all those angular tables with javascript methods, which would be less code, but I'm not sure if that improves readability?